buildlib/az-distro-release.yml

```diff
@@ -67,13 +84,18 @@ jobs:
 
       - bash: |
           set -eEx
-          cd pkg-build
-          echo 10 > debian/compat   # https://www.debian.org/doc/manuals/maint-guide/dother.en.html#compat
-          sed -i 's|./contrib|../contrib|' debian/rules  # Hack
-          dpkg-buildpackage -us -uc
-          find .. -name '*.deb'
-          find .. -name '*.deb' -exec cp {} "../${AZ_ARTIFACT_NAME}" \;
-          dpkg-deb -I "../${AZ_ARTIFACT_NAME}"
+          ./autogen.sh
+          ./contrib/configure-release --with-cuda --with-java=no
```

Comment:
Is it OK `--with-java=no`?

Reply:
Java package gets built by a separate job.